common/src/test/java/io/druid/metadata/DefaultPasswordProviderTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2015 - Yahoo! Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.metadata;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DefaultPasswordProviderTest
+{
+  private static final String pwd = "nothing";
+
+  @Test
+  public void testFromStringPassword() {
+    DefaultPasswordProvider pp = new DefaultPasswordProvider(pwd);
+    Assert.assertEquals(pwd, pp.getPassword());
+  }
+  
+  @Test
+  public void testFromMapPassword() {
+    DefaultPasswordProvider pp = DefaultPasswordProvider.fromString(pwd);
+    Assert.assertEquals(pwd, pp.getPassword());
+  }
+}
```

Comment:
json ser/de is being tested in 
MetadataStorageConnectorConfigTest.java and
MetadataStorageUpdaterJobSpecTest.java

but i'll add another one here as well. thanks.